internal/service/verify/resource_verify_policy.go

```diff
@@ -603,6 +607,17 @@ func (r *VerifyPolicyResource) Schema(ctx context.Context, req resource.SchemaRe
 							int32validator.Between(attrMinRetryAttempts, attrMaxRetryAttempts),
 						},
 					},
+					"verify_aamva": schema.BoolAttribute{
+						Description: "When enabled, the AAMVA DLDV system is used to validate identity documents issued by participating states.",
```

Comment:
It would be good to describe the default behavior in the description since this is computed, something like "true if verify is enabled, null otherwise"